source/PggHostPggSymUnitTest.test.cc

```diff
@@ -0,0 +1,449 @@
+#include "Pgghost.h"
+#include "Pggsym.h"
+
+TEST_CASE("PggSymbiont SetHost, GetHost") {
+
+    emp::Ptr<emp::Random> random = new emp::Random(-1);
+    SymConfigBase config;
+    SymWorld w(*random);
+    double int_val = 1;
+
+    emp::Ptr<Organism> h = new PggHost(random, &w, &config);
+    PGGSymbiont * s = new PGGSymbiont(random, &w, &config, int_val);
+
+    s->SetHost(h);
+
+    REQUIRE(s->GetHost() == h);
+
+}
+
+TEST_CASE("PGGHost SetSymbionts") {

```

Comment:
It doesn't look like you have your own new methods for these, so I'm not sure why you need tests for them if you are just using the Host and Symbiont methods